### PR TITLE
(cli) add studio commands

### PR DESCRIPTION
**Pull request by joewagner**
In order to combine our CLIs we decided to add the studio cli commands as a sub command.  This is accomplished by simply importing the studio cli code and loading the commands under `studio` 

Related to: https://github.com/tablelandnetwork/studio/pull/241

fixes: https://linear.app/tableland/issue/ENG-713/move-all-existing-studio-cli-commands-to-a-sub-command-in-the-original

### CODE REVIEW

File: packages/cli/package.json [hunk 1]
```diff
@@ -52,6 +52,7 @@
     "@tableland/node-helpers": "^0.0.2",
     "@tableland/sdk": "^6.0.0",
     "@tableland/sqlparser": "^1.3.0",
+    "@tableland/studio-cli": "^0.1.0-pre.3",
```
Comment (joewagner) [inline packages/cli/package.json:55]: need to update versions once PRs are approved. https://github.com/tablelandnetwork/studio/pull/241

File: packages/cli/src/commands/studio.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+export const builder: (yargs: Argv<Record<string, unknown>>) => void =
+  function (yargs) {
+    for (const cmnd of studioCommands) {
+      yargs
+        .command(
+          cmnd.command,
+          cmnd.desc,
+          typeof (cmnd as any).builder !== "undefined"
+            ? (cmnd as any).builder
```
Comment (joewagner) [inline packages/cli/src/commands/studio.ts:19]: I spent quite a bit of time trying to get the types to work here, but couldn't find a way. The main [commands call](https://github.com/tablelandnetwork/tableland-js/blob/main/packages/cli/src/cli.ts#L59) is cast to `any` too, so maybe this is ok?

File: packages/sdk/src/validator/client/validator.ts [hunk 3]
```diff
@@ -22,9 +22,14 @@ export interface paths {
   "/query": {
     /**
      * Query the network 
-     * @description Returns the results of a SQL read query against the Tabeland network
+     * @description Returns the results of a SQL read query against the Tableland network
```
Comment (joewagner) [inline packages/sdk/src/validator/client/validator.ts:25]: This whole file is automatically generated from docs, and the task had not been run recently.  Not related to the PR but seemed best to leave this here.

**Review by dtbuchholz — APPROVED** [review]
nice, works great 🚀

**Pull request merged**